rpc/Cargo.toml

```diff
@@ -9,6 +9,9 @@ thiserror = "1.0.40"
 jsonrpsee = { version = "0.20", features = ["client-core", "macros"] }
 celestia-types = { workspace = true }
 serde = { version = "1.0.188", features = ["derive"] }
+libp2p = "0.52.3"
```

Comment:
what'd you say to having libp2p and p2p module behind a feature flag? It's probably not so common case, especially guarded with ADMIN permission level, so I think it can be a default but still a feature